curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fabf0dd4959805dbae09e6.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-31
 
 Add another `label` after the first, with the text `Input your age (years): `. Then, nest an `input` with the `type` of `number`.
 
-As we do not want users under the age of 13 to register, add a `min` attribute to the `input` with a value of `13`. Also, we can probably assume users over the age of 120 will not register; add a `max` attribute with a value of `120`.
+Next, Add a `min` attribute to the `input` with a value of `13` because users under the age of 13 should not register. Also, it can be probably assumed that users over the age of 120 will not register; add a `max` attribute with a value of `120`.
```

Comment:
```suggestion
Next, add a `min` attribute to the `input` with a value of `13` because users under the age of 13 should not register. Also, it can be probably assumed that users over the age of 120 will not register; add a `max` attribute with a value of `120`.
```

No capitalisation after a comma :)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
[@naomi-lgbt](https://github.com/naomi-lgbt), I appologize that I've overlooked it. Thanks for your suggestion.